lib/web_ui/test/canvaskit/text_test.dart

```diff
@@ -54,6 +54,21 @@ void testMain() {
         expect(paragraph, isNotNull);
       }
     });
+
+    test('works for https://github.com/flutter/flutter/issues/78550', () {
```

Comment:
Love linking to the regression, but shouldn't the name of the test be a bit more descriptive?

Reply:
Done